src/Npc.js

```diff
@@ -99,6 +99,27 @@ class Npc extends Character {
       this.addItem(newItem);
     });
 
+    // Hydrate equipment
+    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
```

Comment:
`defaultEquipment` should always an object (`{}`), it should never be an array or a `Map` so this is not necessary.

Reply:
I'm still not sure this `if` is necessary since `defaultEquipment` isn't an array and will never be a `Map`

Reply:
So, should I just remove that whole if clause?  the instanceof Map part was already there from Player.js, and the equipment.length was there because I was still confusedly referencing equipment instead of defaultEquipment.

```
    // Hydrate equipment
    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
      const eqDefs = this.defaultEquipment;
      this.defaultEquipment = new Map();
      for (const [slot, itemRef] of Object.entries(this.defaultEquipment)) {
        const itemDef = eqDefs[slot];
        try {
          let newItem = state.ItemFactory.create(state.AreaManager.getAreaByReference(itemDef), itemDef);
         // newItem.initializeInventory(itemDef.inventory);
          newItem.hydrate(state);
          state.ItemManager.add(newItem);
          newItem.isEquipped = true;
          this.equip(newItem, slot);
        } catch (e) {
          Logger.error(e.message);
        }
      }
    } else {
      this.defaultEquipment = new Map();
    }
```